csharp/src/Drivers/Apache/Hive2/README.md

```diff
@@ -35,11 +35,15 @@ but can also be passed in the call to `AdbcDatabase.Connect`.
 | `username`             | The user name used for basic authentication | |
 | `password`             | The password for the user name used for basic authentication. | |
 | `adbc.hive.data_type_conv` | Comma-separated list of data conversion options. Each option indicates the type of conversion to perform on data returned from the Hive server. <br><br>Allowed values: `none`, `scalar`. <br><br>Option `none` indicates there is no conversion from Hive type to native type (i.e., no conversion from String to Timestamp for Apache Hive over HTTP). Example `adbc.hive.conv_data_type=none`. <br><br>Option `scalar` will perform conversion (if necessary) from the Hive data type to corresponding Arrow data types for types `DATE/Date32/DateTime`, `DECIMAL/Decimal128/SqlDecimal`, and `TIMESTAMP/Timestamp/DateTimeOffset`. Example `adbc.hive.conv_data_type=scalar` | `scalar` |
-| `adbc.hive.tls_options` | Comma-separated list of TLS/SSL options. Each option indicates the TLS/SSL option when connecting to a Hive server. <br><br>Allowed values: `allow_self_signed`, `allow_hostname_mismatch`. <br><br>Option `allow_self_signed` allows certificate errors due to an unknown certificate authority, typically when using a self-signed certificate. Option `allow_hostname_mismatch` allow certificate errors due to a mismatch of the hostname. (e.g., when connecting through an SSH tunnel). Example `adbc.hive.tls_options=allow_self_signed` | |
 | `adbc.hive.connect_timeout_ms` | Sets the timeout (in milliseconds) to open a new session. Values can be 0 (infinite) or greater than zero. | `30000` |
 | `adbc.apache.statement.batch_size` | Sets the maximum number of rows to retrieve in a single batch request. | `50000` |
 | `adbc.apache.statement.polltime_ms` | If polling is necessary to get a result, this option sets the length of time (in milliseconds) to wait between polls. | `500` |
 | `adbc.apache.statement.query_timeout_s` | Sets the maximum time (in seconds) for a query to complete. Values can be 0 (infinite) or greater than zero. | `60` |
+| `adbc.apache.ssl` | If ssl needs to enabled or not. One of `True`, `False` | `False` |
+| `adbc.apache.enable_server_certificate_validation` | If ssl server certificate validation needs to enabled or not. One of `True`, `False`. If set to False, all certificate validation errors are ignored | `True` |
```

Comment:
The option name `enable_server_certificate_validation` implies that we're opting into enabling it. I think that it's more semantically clear to have an option `disable_server_certificate_validation` with a default value of false to give the correct impression that the settings are secure by default and that you have to opt into less security.